css/style.css

```diff
@@ -30,6 +30,7 @@
   padding-top: 24px;
   text-decoration-line: underline;
   color: #fff;
+  font-variant: small-caps;
```

Comment:
I would suggest styling the links without an underline here 👍🏽 because it would mess up the look and feel of your design.